src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -383,6 +389,26 @@ trait BeamHelper extends LazyLogging {
     }), scala.concurrent.duration.Duration.Inf)
   }
 
+  def writeScenarioPrivateVehicles(scenario: MutableScenario, beamServices: BeamServices) = {
+    val csvWriter: FileWriter = new FileWriter("householdVehicles.csv", true)
+    try {
+      csvWriter.write("vehicleId,vehicleType,householdId\n")
+      scenario.getHouseholds.getHouseholds.values.asScala.foreach { householdId =>
+        householdId.getVehicleIds.asScala.foreach { vehicle =>
+          beamServices
+            .privateVehicles
+            .get(vehicle)
+            .map(
+              v => v.id.toString + "," + v.beamVehicleType.id.toString + "," + householdId.getId.toString + "\n"
+            )
+            .map((csvWriter.write(_)))
```

Comment:
When the mapped variables match the parameter list then you can just pass the function reference:

    .map(csvWriter.write)